profile/ro-crate-metadata.json

```diff
@@ -0,0 +1,350 @@
+      "name": "PropertyValue"
+    },
+    {
+      "@id": "https://schema.org/DefinedTerm",
+      "@type": "DefinedTerm",
+      "name": "DefinedTerm"
+    },
+    {
+      "@id": "https://schema.org/Person",
+      "@type": "DefinedTerm",
+      "name": "Person"
+    },
+    {
+      "@id": "https://schema.org/ScholarlyArticle",
+      "@type": "DefinedTerm",
+      "name": "ScholarlyArticle"
+    },
+    {
+      "@id": "https://schema.org/Comment",
+      "@type": "DefinedTerm",
+      "name": "Comment"
+    },
+    {
+      "@id": "https://schema.org/MediaObject",
+      "@type": "DefinedTerm",
+      "name": "MediaObject"
+    },
+    {
+      "@id": "https://bioschemas.org/types/Sample/0.3-DRAFT",
+      "@type": "DefinedTerm",
+      "name": "Sample"
+    },
+    {
+      "@id": "https://schema.org/Organization",
+      "@type": "DefinedTerm",
+      "name": "Organization"
+    },
+    {
+      "@id": "isa_ro_crate_mapping.md",
+      "@type": "File",
+      "name": "ISA RO-Crate ISA Tab/Json mapping guidance",
+      "encodingFormat": "text/markdown",
+      "about": {
+        "@id": "https://github.com/nfdi4plants/isa-ro-crate-profile/tree/1.0.0-draft.2/profile"
+      }
+    },
+    {
+      "@id": "https://arc-rdm.org/details/documentation-principle/",
+      "@type": "File",
+      "name": "ISA RO-Crate Annotation Modelling guidance",
+      "encodingFormat": "text/html",
+      "about": {
+        "@id": "https://github.com/nfdi4plants/isa-ro-crate-profile/tree/1.0.0-draft.2/profile"
+      }
+    },
+    {
+      "@id": "https://git.nfdi4plants.org/venn/Ru_ChlamyHeatstress/-/package_files/35699/download",
+      "@type": "File",
+      "name": "ISA RO-Crate Example",
+      "encodingFormat": "application/json",
+      "about": {
+        "@id": "https://github.com/nfdi4plants/isa-ro-crate-profile/tree/1.0.0-draft.2/profile"
+      }
+    },
+    {
+      "@id": "#hasSpecification",
+      "@type": "ResourceDescriptor",
+      "hasRole": {
+        "@id": "http://www.w3.org/ns/dx/prof/role/specification"
+      },
+      "hasArtifact": {
+        "@id": "isa_ro_crate.md"
+      }
+    },
+    {
+      "@id": "#hasGuidance",
+      "@type": "ResourceDescriptor",
+      "hasRole": {
+        "@id": "http://www.w3.org/ns/dx/prof/role/guidance"
+      },
+      "hasArtifact": [
+        {
+          "@id": "isa_ro_crate_mapping.md"
+        },
+        {
+          "@id": "https://arc-rdm.org/details/documentation-principle/"
+        }
+      ]
+    },
+    {
+      "@id": "#hasExample",
+      "@type": "ResourceDescriptor",
+      "hasRole": {
+        "@id": "http://www.w3.org/ns/dx/prof/role/example"
+      },
+      "hasArtifact": {
+        "@id": "https://git.nfdi4plants.org/venn/Ru_ChlamyHeatstress/-/package_files/35699/download"
+      }
+    },
+    {
+      "@id": "https://github.com/nfdi4plants/isa-ro-crate-profile/tree/1.0.0-draft.2/profile",
+      "@type": [
+        "Dataset",
+        "Profile"
+      ],
```

Comment:
Add `version` and `keywords` - both optional, but useful (keywords especially beneficial for the [profile registry](https://profiles.ro-crate.org/en/profiles/faceted-search/table?page=0), you can see some examples there)